docs/graphql/manual/auth/authentication/jwt.rst

```diff
@@ -92,7 +92,8 @@ etc.) JWT claims, as well as Hasura specific claims inside a custom namespace
 (or key) i.e. ``https://hasura.io/jwt/claims``.
 
 The ``https://hasura.io/jwt/claims`` is the custom namespace where all Hasura
-specific claims have to be present. This value can be configured in the JWT
+specific claims have to be present. This value can be configured using
```

Comment:
This feels is misleading (also in the paragraph above). It should say `https://hasura.io/jwt/claims` something along the lines of `https://hasura.io/jwt/claims` is the default namespace when `claims_namespace` or `claims_namespace_path` are not provided.

Reply:
NVM, this was explaining the example JWT above.